instrumentation/kotlinx-coroutines/javaagent/gradle.properties

```diff
@@ -0,0 +1 @@
+kotlin.stdlib.default.dependency=false
```

Comment:
Who uses this property?

Reply:
kotlin gradle plugin, can check out from the link